.goreleaser.yml

```diff
@@ -49,5 +49,5 @@ dockers:
       - "--platform=linux/amd64"
     dockerfile: Dockerfile.releaser
     image_templates:
-      - "libsv/p4:latest"
-      - "libsv/p4:{{.Version}}"
+      - "libsv/dp3:latest"
```

Comment:
@theflyingcodr you think this will work if all I did was change it from p4 to dp3? what else do we need to do to get the docker images pushed to docker hub?

Reply:
should this not be dpp?